consensus/src/execution_ctx.rs

```diff
@@ -532,3 +558,16 @@ impl<'a, T: Operations + 'static, DB: Database> ExecutionCtx<'a, T, DB> {
         self.iter_ctx.get_generator(self.iteration)
     }
 }
+
+#[inline(always)]
+fn log_msg(event: &str, src: &str, msg: &Message) {
+    debug!(
+        event,
+        src,
+        msg_step = msg.get_step(),
```

Comment:
I think the step is not necessary since we have the topic.
To be removed in follow-up PR